ecc/bls12-377/twistededwards/point.go

```diff
@@ -481,26 +482,25 @@ func (p *PointExtended) FromAffine(p1 *PointAffine) *PointExtended {
 // dedicated addition
 // https://hyperelliptic.org/EFD/g1p/auto-twisted-extended-1.html#addition-add-2008-hwcd-4
```

Comment:
after the change, the url links to a different formula. Correct link: https://hyperelliptic.org/EFD/g1p/auto-twisted-extended.html#addition-add-2008-hwcd-2